renovate.json

```diff
@@ -1,18 +1,17 @@
 {
   "labels": ["renovate"],
   "extends": ["config:base"],
+  "schedule": ["every weekend"],
+  "branchConcurrentLimit": 20,
+  "dependencyDashboard": true,
+  "major": {
+    "dependencyDashboardApproval": true
+  },
   "packageRules": [
     {
       "matchUpdateTypes": ["minor", "patch", "pin", "digest"],
       "matchCurrentVersion": "!/^0/",
-      "automerge": true,
-      "rebaseWhen": "behind-base-branch"
-    },
-    {
-      "matchUpdateTypes": ["major"],
-      "baseBranches": ["renovate-major"],
-      "automerge": false,
-      "rebaseWhen": "behind-base-branch"
+      "automerge": true
```

Comment:
Does it default to `behind-base-branch` when we set `automerge` to true?

Reply:
It's on `auto` mode. Also, we can tick the checkbox on the PR OP for on-demand rebase. I tested it.